cucumber/features/devLoginPup.feature

```diff
@@ -0,0 +1,9 @@
+Feature: TTA Smarthub HSES login
+ 
+    Scenario: Welcome page is displayed
+        When visiting https://tta-smarthub-dev.app.cloud.gov
```

Comment:
Is there a way to template out the URL? I imagine we will want to hit a locally running server in the CI pipeline.

Reply:
Does Ryan want us testing dev? For the OWASP scan he wanted us to be testing a locally running version of the application (so we would be testing the new code a PR introduces) instead of testing dev or staging.